src/steps/02-EvaluationCriteria/Exceptions.vue

```diff
@@ -117,6 +83,57 @@ export default class Exceptions extends Mixins(SaveOnLeave) {
   private selectedException 
     = AcquisitionPackage.fairOpportunity?.exception_to_fair_opportunity as string;
 
+  /**
+   * Returns the type value for the ATATAlert component, based
+   * on the selected exception option.
+   */
+  get evalAlertType(): string {
+    return  this.selectedException !== "" && this.selectedException !== "NO_NONE" ?
+      "warning" : "info";
+  }
+
+  /**
+   * Returns the template content for the ATATAlert component, based
+   * on the selected exception option.
+   */
+  get evalAlertContent(): string {
+    if (this.selectedException === 'YES_FAR_16_505_B_2_I_B' ||
+        this.selectedException === 'YES_FAR_16_505_B_2_I_C') {
+      return `
+      <p>
+        Your final acquisition package will require a
+        <strong>
+          Justification & Approval (J&A)
+        </strong>
+        . We’ll help you complete all of your required justification documentation.
+      </p> `;
+    } else if (this.selectedException === 'YES_FAR_16_505_B_2_I_A') {
+      return `
+      <p>
+        Your final acquisition package will require a
+        <strong>
+          Justification & Approval (J&A)
+        </strong>
+        . We’ll help you complete all of your required justification documentation.
+      </p>
```

Comment:
I will do the change as you suggested but the risk of a copy change is less the code complexity and maintainability in the long run. Generally, it will seem like the duplication is obvious but as we go along and the requirements evolve, this simple duplication will end up being a complex logic and hard to understand.

Reply:
Here is how the complexity starts..For example if the requirements change such that "Do not display the below in bold for the second option". Then an other and and an other as requirements evolve. Within no time this function will become a complex function with lots of edge cases that could fail.
<strong>
          Justification & Approval (J&A)
        </strong>